arch/arm64/boot/dts/freescale/crocodile.dtsi

```diff
@@ -0,0 +1,770 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright 2020 NXP
+ * Copyright (C) 2021 Leica Geosystems AG
+ */
+
+/dts-v1/;
+
+#include <dt-bindings/usb/pd.h>
+#include "imx8mm.dtsi"
+#include "crocodile-pinmap.h"
+
+/ {
+	compatible = "fsl,imx8mm-evk", "fsl,imx8mm";
```

Comment:
Why is `fsl,imx8mm-evk` binding required here? So far, I see no consumer in the Kernel, and it is not even applicable to our setup.

Since we do not include `imx8mm-evk`, I suggest to drop it here.